src/index.css

```diff
@@ -69,3 +70,68 @@ button:focus-visible {
     background-color: #F9F9F9;
   }
 }
+
+h2 {
+  font-size: 36px;
```

Comment:
Please add a `line-height` here and for all texts

Reply:
done